SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/SpatialGDKEditorSchemaGenerator/SpatialGDKEditorSchemaGeneratorTest.cpp

```diff
@@ -1181,6 +1184,13 @@ SCHEMA_GENERATOR_TEST(GIVEN_actor_class_WHEN_generating_schema_THEN_expected_com
 			{
 				return false;
 			}
+
+			// Auth Server Only expected to have not components in schema?
+			if (ComponentType == SCHEMA_AuthServerOnly)
+			{
+				return true;
```

Comment:
this should be a continue, not a return

Reply:
Yeah, wasn't sure of the behavior, hence the (?)! haha So then I'd have something like below, or is the set broken if it is empty?                                                
`
			if (ComponentType != SCHEMA_AuthServerOnly)
			{
				// We should have a class for each type of set
				TestTrue("Set is not empty", DataComponents->ComponentIDs.Num() > 0);
			}`

Reply:
ah, I see I missed creating a new dummy object within SchemaGenObjectstub, will make one and verify